demos/object_detection_demo/jupyter_python/object_detection_demo.ipynb

```diff
@@ -0,0 +1,754 @@
+    "            \"--mo\",\n",
+    "            model_optimizer_path,\n",
+    "            \"--jobs\",\n",
+    "            \"4\",\n",
+    "        ],\n",
+    "        stdout=subprocess.PIPE,\n",
+    "        stderr=subprocess.PIPE,\n",
+    "        shell=False,\n",
+    "        universal_newlines=True,\n",
+    "    )\n",
+    "    if converter_result.returncode == 0:\n",
+    "        print(\n",
+    "            \"Converting models succeeded. Set CONVERT_MODELS to False to save time when you run this notebook again.\"\n",
+    "        )\n",
+    "    else:\n",
+    "        error_message = \"Not all models were converted succesfully. Check the value of `converter_result` for more details. \"\n",
+    "        if \"No module named\" in converter_result.stderr or \"install_prerequisites\" in converter_result.stdout:\n",
+    "            error_message += \"Make sure that the Model Optimizer requirements are installed before running the Model Converter. See the <a href='https://docs.openvinotoolkit.org/latest/installation_guides.html'>OpenVINO installation guide</a> for more information\"\n",
+    "        display(HTML(f'<div class=\"alert alert-warning\" style=\"color:black\"><i><b>Warning: </b>{error_message}</div>'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "national-plaintiff",
+   "metadata": {},
+   "source": [
+    "### Get model info\n",
+    "\n",
+    "The Info Dumper returns information for the Open Model Zoo models. It returns a list of dictionaries with the model name, description, framework, license url, precisions, task type, and the subdirectory for the downloaded model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "naked-iraqi",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "info_command = os.path.join(open_model_zoo_path, \"tools\", \"downloader\", \"info_dumper.py\")\n",
+    "info_result = subprocess.run(\n",
+    "    [\n",
+    "        sys.executable,\n",
+    "        info_command,\n",
+    "        \"--list\",\n",
+    "        models_file,\n",
+    "    ],\n",
+    "    stdout=subprocess.PIPE,\n",
+    "    stderr=subprocess.PIPE,\n",
+    "    shell=False,\n",
+    "    universal_newlines=True,\n",
+    ")\n",
+    "info = json.loads(info_result.stdout)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "tropical-forest",
+   "metadata": {},
+   "source": [
+    "Make a list of models that will be shown as options. By default, only Intel models will be shown. Change this by uncommenting the second line to use all models. You need to make sure that the models are in IR format. This can be done by setting `CONVERT_MODELS` to `True` and running all cells. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "hungarian-opposition",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_names = [model[\"name\"] for model in info if \"intel\" in model[\"subdirectory\"]]\n",
```

Comment:
The condition here is unreliable, because the string "intel" could just be a part of the model name or subdirectory. Moreover, it's indirect. You'd be better off just checking whether the framework is "dldt".

Reply:
Done.